test/functional/Java21Only/src/org/openj9/test/jep442/downcall/UnionStructTests.java

```diff
@@ -21,25 +21,31 @@
  */
 package org.openj9.test.jep442.downcall;
 
-import org.testng.annotations.Test;
 import org.testng.Assert;
-import org.testng.AssertJUnit;
+import org.testng.annotations.Test;
 
 import java.lang.invoke.MethodHandle;
 
 import java.lang.foreign.Arena;
-import java.lang.foreign.Linker;
 import java.lang.foreign.FunctionDescriptor;
 import java.lang.foreign.GroupLayout;
+import java.lang.foreign.Linker;
 import java.lang.foreign.MemoryLayout;
 import java.lang.foreign.MemoryLayout.PathElement;
```

Comment:
Please fix the order of imports and remove unused imports:
* MemoryLayout.PathElement
* UnionLayout
* ValueLayout.ADDRESS

Reply:
> Please correct the order of imports

I don't see there is any misplaced imports (please point it out if I ignored).

> remove unused imports:...

Fixed.